src/wwwroot/partials/settings/dkim-configuration-help.html

```diff
@@ -10,9 +10,7 @@
   <hr class="default" />
 </div>
 <div class="item section--wrapper flex">
-  <h2>
-    {{ 'dkim_configuration' | translate }}
-  </h2>
+  <h3><strong>{{ 'dkim_configuration' | translate }}</strong></h3>
```

Comment:
After a research i found that in the _typography.scss file we only have h1 and h2 as bold, but below of those definitions there is a class named "special" that adds bold and a line-height into any of the other headers, i think you should use that class.